TLM/TLM/Util/LaneUtil.cs

```diff
@@ -0,0 +1,34 @@
+namespace TrafficManager.Util {
+    using TrafficManager.Util.Extensions;
+    public static class LaneUtil {
+        /// <summary>
+        /// gets the index of lane id or -1 if lane was not found
+        /// </summary>
+        public static int GetLaneIndex(uint laneId) {
+            ref NetLane netLane = ref laneId.ToLane();
+            ref NetSegment netSegment = ref netLane.m_segment.ToSegment();
+            var lanes = netSegment.Info?.m_lanes;
+            if (lanes != null) {
+                uint curLaneId = netSegment.m_lanes;
+                for (int laneIndex = 0; laneIndex < lanes.Length && curLaneId != 0; ++laneIndex) {
+                    if (curLaneId == laneId) {
+                        return laneIndex;
+                    }
+                    laneIndex++;
```

Comment:
Did you commit to wrong branch?

https://github.com/CitiesSkylinesMods/TMPE/pull/1272/commits/a4d5a790294b03d9b9c41edb1628b020949860fe

Reply:
Yep it's a bug here

Reply:
Oh... ook @aubergine10 #1275 is targeting this branch, so... confusing but fixed. We just need to merge #1275 first, I think

Reply:
yes that is a bug that I already fix. I am surprised the fix was not uploaded.

Reply:
I had committed the fix to the wrong branch. now is fine.